src/lib/libcore.js

```diff
@@ -2290,6 +2290,7 @@ addToLibrary({
   'element': 'anyfunc'
 });
 `,
+  $wasmTable__postset: () => addAtInit('usedTableLength = wasmTable.length'),
```

Comment:
I think you need to add `$usedTableLength` to `wasmTable__deps`.

Reply:
Oh I see the other discussion.

Reply:
Sure, I just removed it. Let's see if the test in ci/cd again complains about a missing usedTableLength.

Reply:
Ok I have a test environment. But seems impossible (after tracing the calls with logging for wasm allocation) to do the initial setting correctly. I will abandon the idea of usedTableLength and do something else, which is closer to my initial idea, but uses less memory and has some desirable benefits. It will come during the next hours.